vic/drivers/image/src/vic_start.c

```diff
@@ -101,8 +105,14 @@ vic_start(void)
         // get the indices for the active cells (used in reading and writing)
         filter_active_cells = (size_t *) malloc(global_domain.ncells *
                                                 sizeof(size_t));
-        for (i = 0; i < global_domain.ncells; i++) {
-            filter_active_cells[i] = global_domain.locations[i].io_idx;
+        j = 0;
+        for (y = 0, i = 0; y < global_domain.n_ny; y++) {
+            for (x = 0; x < global_domain.n_nx; x++, i++) {
```

Comment:
@bartnijssen and I discussed added one more member to the `domain_struct` to handle the different sizes of the active/total domains. We were thinking it would be good to rename `ncells` to `ncells_active` and add a new member `ncells_total`.  This would keep us from doing two leveled x/y loops like this and would simplify logic in a number of other places. @wietsefranssen, what do you think?